aspnetcore/signalr/troubleshoot.md

```diff
@@ -0,0 +1,91 @@
+---
+title: ASP.NET Core SignalR connection troubleshooting
+author: 
+description: ASP.NET Core SignalR connection troubleshooting.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
```

Comment:
@Rick-Anderson I see your screenshot with the suggested pending changes to add @bradygaster as an author to this doc. Did you mean to submit a new review with those suggestion?

Reply:
same question.